spec/system/task_spec.rb

```diff
@@ -1,24 +1,24 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  let(:project) {create(:project)}
+  let(:task) {create(:task, project_id: project.id)}
```

Comment:
`association`の定義があれば`project_id`を指定する必要はなくなると思います。ご確認ください。
```suggestion
  let(:task) { create(:task) }
```

Reply:
b596171c3e07c3441aff12683d4e85b38b4903c7
`association`を定義することによって`project_id`のコードを削除しました。